packages/contracts-bedrock/test/setup/ForkLive.s.sol

```diff
@@ -155,8 +155,13 @@ contract ForkLive is Deployer {
         address upgrader = proxyAdmin.owner();
         vm.label(upgrader, "ProxyAdmin Owner");
 
-        IOPContractsManager.OpChain[] memory opChains = new IOPContractsManager.OpChain[](1);
-        opChains[0] = IOPContractsManager.OpChain({ systemConfigProxy: systemConfig, proxyAdmin: proxyAdmin });
+        IOPContractsManager.OpChainConfig[] memory opChains = new IOPContractsManager.OpChainConfig[](1);
+        opChains[0] = IOPContractsManager.OpChainConfig({
+            systemConfigProxy: systemConfig,
+            proxyAdmin: proxyAdmin,
+            permissionedDisputeGamePrestateHash: Claim.wrap(bytes32(keccak256("permissionedPrestate"))),
+            permissionlessDisputeGamePrestateHash: Claim.wrap(bytes32(keccak256("permissionlessPrestate")))
+        });
 
         // TODO Migrate from DelegateCaller to a Safe to reduce risk of mocks not properly
         // reflecting the production system.
```

Comment:
I think we need a tracking issue for this, I don't see one in https://github.com/ethereum-optimism/optimism/issues/13069

Reply:
https://github.com/ethereum-optimism/optimism/issues/13993